tests/test_env_checker.py

```diff
@@ -112,3 +112,58 @@ def step(self, action):
     test_env = TestEnv()
     with pytest.raises(AssertionError, match=error_message):
         check_env(env=test_env)
+
+
+class StepCalledAfterEnvTerminatedException(Exception):
+    pass
+
+
+class LimitedStepsTestEnv(gym.Env):
+    metadata = {"render_modes": ["human"]}
+    render_mode = None
```

Comment:
maybe add it as a parameter too?

Reply:
yes, I meant allow `render_mode` as a parameter to the constructor, or remove everything, yes.